### PR TITLE
🥑🤖 v1.3.0 release prep

### PR DESCRIPTION
**Pull request by brycahta**
🥑🤖 Auto-generated PR for v1.3.0 release. Updating release versions in repo.

### CODE REVIEW

**Comment by brycahta** [comment]
[release draft](https://github.com/aws/amazon-ec2-metadata-mock/releases/edit/untagged-fdcb5ad3d1077ac30b96)

File: README.md [hunk 1]
```diff
@@ -183,7 +183,7 @@ local-ipv4
 mac
 network/interfaces/macs/0e:49:61:0f:c3:11/device-number
 network/interfaces/macs/0e:49:61:0f:c3:11/interface-id
-network/interfaces/macs/0e:49:61:0f:c3:11/ipv4-associations/192.0.2.54
```
Comment (pdk27) [inline README.md:186]: oops. looks like we found a bug in the automation script!

File: README.md [hunk 2]
```diff
@@ -410,7 +410,7 @@ TOKEN=`curl -X PUT "localhost:1338/latest/api/token" -H "X-aws-ec2-metadata-toke
 0e:49:61:0f:c3:11
 ```
 
-Requesting a token outside the TTL bounds (between 1-2600 seconds) will return **400 - Bad Request:**
```
Comment (pdk27) [inline README.md:413]: oh no! 
anything like 1.*2 has been replaced :#

Comment (brycahta) [reply README.md]: good catch! fixing manually

**Pull request merged**